static/app/components/contextData/index.tsx

```diff
@@ -30,29 +46,36 @@ function walk({
   meta,
 }: {
   depth: number;
+  jsonConsts: boolean;
+  maxDefaultDepth: number;
+  meta: Record<any, any> | undefined;
+  preserveQuotes: boolean;
+  withAnnotatedText: boolean;
   value?: React.ReactNode;
-} & Pick<
-  Props,
-  'withAnnotatedText' | 'preserveQuotes' | 'jsonConsts' | 'meta' | 'maxDefaultDepth'
->) {
+}) {
   let i = 0;
 
   const children: React.ReactNode[] = [];
 
   if (value === null) {
     return (
       <span className="val-null">
-        <AnnotatedText value={jsonConsts ? 'null' : 'None'} meta={meta?.[''] ?? meta} />
+        <ContextValue
+          value={jsonConsts ? 'null' : 'None'}
```

Comment:
@malwilley literal 'null' and 'None` strings seem janky

Reply:
I am changing this in a future PR, this is the current behavior